common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -368,14 +407,8 @@ def validate(self):
 
     @property
     def non_editable_metadata_fields(self):
-        non_editable_fields = super(ConditionalDescriptor, self).non_editable_metadata_fields
+        non_editable_fields = super(ConditionalBlock, self).non_editable_metadata_fields
         non_editable_fields.extend([
-            ConditionalDescriptor.due,
-            ConditionalDescriptor.is_practice_exam,
-            ConditionalDescriptor.is_proctored_enabled,
-            ConditionalDescriptor.is_time_limited,
-            ConditionalDescriptor.default_time_limit_minutes,
-            ConditionalDescriptor.show_tag_list,
-            ConditionalDescriptor.exam_review_rules,
```

Comment:
These were for fields from `ProctoringFields` mixin which is no longer being inherited from.